tests/test-options.ts

```diff
@@ -0,0 +1,98 @@
+import { test as base } from '@playwright/test';
+import { PageManager } from '../pages/pageManager';
+import { RandomDataHelper } from '../pages/randomDataHelper'
+import { APIHelper} from '../pages/apiHelper'
+
+
+export type TestOptions = {
+    pageManager: PageManager
+    newOwnerFixture: [{
+        'firstName': string,
+        'lastName': string,
+        'address': string,
+        'city': string,
+        'telephone': string,
+        'petName': string,
+        'petBirthdate': Date,
+        'petType': string,
+        'visitDate': Date,
+        'visitDescription': string
+    }]
+}
+
+function setOwnerPetAndVisitVariables(){
+    const randomDataHelper = new RandomDataHelper()
+    const testVars = {
+        'firstName': randomDataHelper.getFirstName(),
+        'lastName': randomDataHelper.getLastName(),
+        'address': randomDataHelper.getAddress(),
+        'city': randomDataHelper.getCity(),
+        'telephone': randomDataHelper.getPhone(),
+        'petName': randomDataHelper.getPetName(),
+        'petBirthdate': randomDataHelper.getPetBirthdate(),
+        'petType': randomDataHelper.getPresetPetType(),
+        'visitDate': new Date(),
+        'visitDescription': randomDataHelper.getVisitDescription()
+
+    }
+    return testVars
+}
+
+export const test = base.extend<TestOptions>({
+
+    pageManager: [async ({ page }, use) => {
+        const pm = new PageManager(page)
+
+        await use(pm)
+
+    }, {auto: true}],
+
+    newOwnerFixture: async ({ pageManager, request }, use) => {
+        const testVars = setOwnerPetAndVisitVariables()
+
+        const firstName = testVars.firstName
+        const lastName = testVars.lastName
+        const address = testVars.address
+        const city = testVars.city
+        const telephone = testVars.telephone
+        const petName = testVars.petName
+        const petBirthdate = testVars.petBirthdate
+        const petType = testVars.petType
+        const visitDate = testVars.visitDate
+        const visitDescription = testVars.visitDescription
```

Comment:
All these constants can be replaced with the object that you was creating in function setOwnerPetAndVisitVariables(), just do everything here.